client/state/jetpack-settings/settings/test/reducer.js

```diff
@@ -84,9 +84,9 @@ describe( 'reducer', () => {
 					12345678: SETTINGS_FIXTURE[ siteId ]
 				},
 				action = {
-					type: JETPACK_SETTINGS_RECEIVE,
```

Comment:
The test for `JETPACK_SETTINGS_RECEIVE` should still remain; I suggest introducing an additional similar test for `JETPACK_SETTINGS_UPDATE_SUCCESS`, keeping the existing test for `JETPACK_SETTINGS_RECEIVE`.

You may ask why? Well, because the reducer should be updating the settings in the 2 cases:

* After fetching the settings
* After updating the settings (which also returns the updated settings)

So let's keep these 2 cases, and add appropriate tests for both of them.